src/App.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable import/extensions */
```

Comment:
You should not use `eslint-disable` at all. Try to fix or reach out your team-members to solve the issues.